py/dml/dmlparse.py

```diff
@@ -2632,6 +2659,12 @@ def ident(t):
 def ident(t):
     t[0] = t[1]
 
+def ident_enforce_not_discard_ref(site, ident):
+    if (str(ident) == '_'
```

Comment:
`str` should be redundant?

Reply:
_Probably._ I did it because `reserved` also does it.

Reply:
oh, that's probably python2 legacy, the `unicode` vs `str` vs `bytes` discrepancy. Everything is strings now (unicode errors are caught in `toplevel`), so this can be removed in `reserved` too.

Reply:
Nice